Applications/FontEditor/main.cpp

```diff
@@ -60,12 +60,12 @@ int main(int argc, char** argv)
     args_parser.add_positional_argument(path, "The font file for editing.", "file", Core::ArgsParser::Required::No);
     args_parser.parse(argc, argv);
 
-    RefPtr<Gfx::Font> edited_font;
+    RefPtr<Gfx::BitmapFont> edited_font;
     if (path == nullptr) {
         path = "/tmp/saved.font";
-        edited_font = Gfx::FontDatabase::default_font().clone();
+        edited_font = static_cast<Gfx::BitmapFont*>(&Gfx::FontDatabase::default_font().clone().leak_ref());
```

Comment:
Is there a nicer way to do this?

Reply:
Make `clone()` return a mutable reference instead maybe?
const_cast is fine too though imho

Reply:
Can you give an example? Maybe I'm a bit daft.
clone() returns a `NonnullRefPtr<Font>` but I want to convert it to a `RefPtr<BitmapFont>`

Reply:
Can you use `static_ptr_cast` for this?

Reply:
Indeed, I can. Didn't know that was a thing :+1:

Reply:
Wow, I somehow misread the entire thing here :/

Reply:
Happens to the best of us :smile: